lib/plugins/aws/provider/awsProvider.js

```diff
@@ -348,6 +348,13 @@ class AwsProvider {
     const stageUpper = this.getStage() ? this.getStage().toUpperCase() : null;
 
     // add specified credentials, overriding with more specific declarations
+    try {
+      impl.addProfileCredentials(result, 'default');
+    } catch (err) {
+      if (err.message !== 'Profile default does not exist') {
```

Comment:
Note for a future (as it's more a general problem not really specific to this PR).

It'll be way better if we accompany errors with error codes and programmatically rely on that for processing.

Having that in this case we will confirm on error via e.g. `error.code !== 'UNKNOWN_PROFILE'`

Reply:
good idea. add at ticket?

Reply:
Yes I'll open an issue for that